src/AboutPage/About.tsx

```diff
@@ -2,7 +2,46 @@ import React from 'react'
 
 function About() {
   return (
-    <div>About</div>
+    <body>
```

Comment:
This should not be a body tag. `<body>` is used to contain all the content of the page, which is done in `index.html`. Change this to a React fragment (`<>`).